terms/PsychENCODESpecific/study.json

```diff
@@ -0,0 +1,125 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id":"sysbio.PECSPecific-annotations.study-0.0.1",
+    "description": "Study",
```

Comment:
The general purpose 'study' key has an empty string for the description. I would either copy that and have just `"description": ""` or we should update the general purpose study key to have the same description as this one.